streamwebs_frontend/streamwebs_frontend/settings.py

```diff
@@ -129,12 +129,8 @@
 
 STATIC_URL = '/static/'
 
-# Temporary list of schools
-SCHOOL_CHOICES = (
-    ('a', 'School A'),
-    ('b', 'School B'),
-    ('c', 'School C'),
-)
+# List of schools
```

Comment:
One last note on this, sorry but I should have mentioned it before - empty values in the setting file should document their format, so people will know what to fill them with.